content/bd/0002-sql.md

```diff
@@ -580,9 +580,9 @@ SELECT product_code, SUM(purchase_quantity) AS quantity
   GROUP BY product_code;
 
 --  product_code | quantity
--- --------------+----------
---  222222       |        9
---  111111       |        7
+-- --------------+-------------------
+--  222222       | 9
+--  111111       | 7
```

Comment:
```suggestion
-- --------------+----------
--  222222       |        9
--  111111       |        7
```
I'm reverting this back to the original content since this is literally the output from PostgreSQL.
Number columns (`quantity` in this case) are right-aligned while text columns (`product_code`, even though I just used numbers in the examples) are left-aligned.